lib/deimos/active_record_consume/batch_consumption.rb

```diff
@@ -41,9 +42,17 @@ def consume_batch(payloads, metadata)
               uncompacted_update(messages)
             end
           end
+          process_valid_records(@valid_active_records)
```

Comment:
Is there a reason this also can't be a notification?

Reply:
This could also be a notification